frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_detail/widgets/mobile_field_name_text_field.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:appflowy/plugins/database_view/application/field/field_editor_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class MobileFieldNameTextField extends StatefulWidget {
+  const MobileFieldNameTextField({
+    this.text,
+    super.key,
+    this.textEditingController,
+  });
+
+  final String? text;
+  final TextEditingController? textEditingController;
```

Comment:
Is it unused?

Reply:
I deleted `textEditingController` here, and used a TextEditingController internally (dispose it)